.github/workflows/docker-build-deploy.yml

```
@@ -31,19 +31,36 @@ jobs:
 
     steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
       - name: Download Docker Image
         uses: actions/download-artifact@v3
         with:
           name: telegram_bot_image
           path: telegram_bot_image.tar.gz
 
+<<<<<<< Updated upstream
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 43: could not find expected ':'

(syntax-check)